src/collection-browser.ts

```diff
@@ -1173,6 +1173,10 @@ export class CollectionBrowser
       width: 18rem;
       padding-right: 12px;
       padding-right: 1rem;
+      max-height: 120vh;
```

Comment:
If you set 120vh max-height, it can't see and use checkbox at bottom if they are more then my screen height. mean it can't scroll facets to select them. but when I use 100vh, I can scroll easily to navigate on those items in facets.

https://user-images.githubusercontent.com/50868725/177097305-fcdfebaf-b184-4818-ba4b-73f7ff221f99.mp4